projects/plugins/jetpack/modules/woocommerce-analytics/classes/class-jetpack-woocommerce-analytics-universal.php

```diff
@@ -470,10 +470,10 @@ public function remove_from_cart_via_quantity() {
 	 */
 	private function get_inner_blocks( $inner_blocks ) {
 		$block_names = array();
-		if ( $inner_blocks['blockName'] ) {
+		if ( isset( $inner_blocks['blockName'] ) && $inner_blocks['blockName'] ) {
```

Comment:
What do you think about doing something like this instead?

```suggestion
		if ( ! empty( $inner_blocks['blockName'] ) ) {
```